src/components/Tokens/TokenTable/TokenRow.tsx

```diff
@@ -50,15 +49,17 @@ const StyledTokenRow = styled.div<{
 }>`
   background-color: transparent;
   display: grid;
-  font-size: 15px;
+  font-size: 16px;
   grid-template-columns: ${({ favoriteTokensEnabled }) =>
     favoriteTokensEnabled ? '1fr 7fr 4fr 4fr 4fr 4fr 5fr 1.2fr' : '1fr 7fr 4fr 4fr 4fr 4fr 5fr'};
-  height: 60px;
   line-height: 24px;
   max-width: ${MAX_WIDTH_MEDIA_BREAKPOINT};
   min-width: 390px;
-  padding-top: ${({ first }) => (first ? '4px' : '0px')};
-  padding-bottom: ${({ last }) => (last ? '4px' : '0px')};
+  ${({ first, last }) => css`
+    height: ${first || last ? '72px' : '64px'};
```

Comment:
would recommend setting the height and then just adjusting the positioning within the container using `align-items` based on first/last

Reply:
the issue is the first and last aren't actually centered, there's a little more spacing above/below in the design

Reply:
then you could calculate the height from `BASE_HEIGHT` + `PADDING` or something. so its clear what the 8px diff comes from